backend-zs/server.js

```diff
@@ -23,4 +23,7 @@ app.use(logger('dev'));
 app.listen(API_PORT, () => console.log(`LISTENING ON PORT ${API_PORT}`));
 
 var userRoutes = require("./routes/user.routes");
-app.use('/api/', userRoutes());
\ No newline at end of file
+var storyRoutes = require('./routes/story.routes')
+
+app.use('/api/', userRoutes());
+app.use('/story/', storyRoutes())
```

Comment:
I think we need to prefix all exposed endpoints with `api`. Also, I'd suggest using the endpoint `get /api/stories/` instead of `/story/all` in order to be consistent with the users endpoints .